bears/general/IndentationBear.py

```diff
@@ -46,8 +47,12 @@ def run(self,
         :param tab_width:
             No. of spaces to insert for indentation.
             Only Applicable if use_spaces is False.
+        :param coalang_dir:
+            Full path of external directory containing the coalang
+            file for language.
         """
-        lang_settings_dict = LanguageDefinition(language)
+        lang_settings_dict = LanguageDefinition(
+                            language, coalang_dir=coalang_dir)
```

Comment:
i thought the external coalang helps in the testing of the commit so it might be logical that it can go in this commit? i have nothing against it going into a new commit though :P